webCommander/centralHub/models.py

```diff
@@ -27,4 +27,17 @@ class Meta:
         unique_together = [('twitter_user','twitter_text')]
 
     def __str__(self):
-        return f'{self.twitter_user.twitter_handle} Posts'
\ No newline at end of file
+        return f'{self.twitter_user.twitter_handle} Posts'
+
+class SpotifyArtistInfo(models.Model):
```

Comment:
As discussed, let's make this a 1-to-many: artist->albums

Reply:
done, just need to populate.